lib/sycamore/sycamore/transforms/base.py

```diff
@@ -193,3 +193,29 @@ def _update_lineage(cls, from_docs, to_docs):
         to_ids = [d.lineage_id for d in to_docs]
 
         return [MetadataDocument(lineage_links={"from_ids": from_ids, "to_ids": to_ids})]
+
+
+class Composite(UnaryNode):
```

Comment:
maybe `CompositeTransform` or `CompositeNode`? idk, just feels a little funny next to `BaseMapTransform` but fine if you wanna keep it

Reply:
done.